docs/shared/recipes/plugins/create-install-package.md

```diff
@@ -0,0 +1,159 @@
+# Creating an Install Package
+
+{% youtube
+src="https://www.youtube.com/embed/ocllb5KEXZk"
+title="Build your own CLI"
+width="100%" /%}
+
```

Comment:
It would be good to have a few sentences as an intro here.  The title is "Creating an Install Package" and you immediately start talking about a preset.  My first thought as the reader is "did I land on the wrong page?"